xml/acls-2.1.rng

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
The README is somewhat wrong on one point -- the file name needs to increment the highest number of any schema file, not just the file being edited, so this file should be acls-3.8.rng. Please correct the README too ...